shell/platform/common/cpp/client_wrapper/testing/stub_flutter_api.h

```diff
@@ -38,6 +38,13 @@ class StubFlutterApi {
                              const uint8_t* message,
                              const size_t message_size) {}
 
+  // Called for FlutterDesktopMessengerSendWithReply.
+  virtual void MessengerSend(const char* channel,
```

Comment:
Why not MessengerSendWithReply? IIRC I was trying to use a predicable naming scheme here relative to the C API names.

Reply:
No good reason. MessengerSendWithReply makes more sense. Done.